index.js

```diff
@@ -91,85 +53,102 @@ const updateDeploymentCommentBody = async (repoOwner, repoName, comment, github)
-// main function for setting the URLs
+// appveyor utility functions
 ///////////////////////////////////////////////
-const setDeployUrl = async (github, githubStatusPayload, getOs, getPassedJobs, getLog) => {
-  application.log("status context: " + githubStatusPayload.context)
-  if(githubStatusPayload.context.includes("pr")){
-    const { passedJobs, prNumber } = await getPassedJobs(
-      githubStatusPayload.target_url,
-      githubStatusPayload.repository.owner.login,
-      githubStatusPayload.repository.name
-    )
-    if(passedJobs.length > 0){
-      const comment = await getDeploymentComment(
-        githubStatusPayload.repository.owner.login,
-        githubStatusPayload.repository.name,
-        prNumber,
-        github
-      )
-      if(!comment) {
-        application.log("Couldn't find our comment, aborting")
-        return
-      }
-      for(let i = 0; i < passedJobs.length; i++) {
-        const job = passedJobs[i]
-        const log = await getLog(job)
-        updateCommentUrlFromLog(getOs(job), log, comment)
-      }
-      updateDeploymentCommentBody(
-        githubStatusPayload.repository.owner.login,
-        githubStatusPayload.repository.name,
-        comment,
-        github
-      )
-    }
-  }
+const getPrNumberFromAppveyor = async (repositoryOwner, repositoryName, buildId) => {
+  const response = await request(`https://ci.appveyor.com/api/projects/${repositoryOwner}/${repositoryName.toLowerCase()}/builds/${buildId}`);
+  const builds = JSON.parse(response)
+  return builds.build.pullRequestId
 }
 
 ///////////////////////////////////////////////
-// main function for creating the comment
+// testing link functions
 ///////////////////////////////////////////////
-const createPrCommentForUs = async (github, payload) => {
-  if(payload.action !== "opened") {
-    return
+const translatePlatform = platform => {
+  if(platform === "macos"){
+    return "osx"
   }
-  application.log("Creating new comment for our use.")
-  const body = "Hey there! Thanks for helping Mudlet improve. :star2:\n\n" +
-          "## Test versions\n\n" +
-          "You can directly test the changes here:\n" +
-          "- linux: (download pending, check back soon!)\n" +
-          "- osx: (download pending, check back soon!)\n" +
-          "- windows: (download pending, check back soon!)\n\n" +
-          "No need to install anything - just unzip and run.\n" +
-          "Let us know if it works well, and if it doesn't, please give details.\n" +
-          (payload.pull_request.title === "New Crowdin translations"
-          ? "\n" +
-            "## Translation stats\n\n" +
-            "calculation pending, check back soon!\n\n"
-          : "")
-  await github.issues.createComment({
-    owner: payload.repository.owner.login,
-    repo: payload.repository.name,
-    number: payload.number,
-    body: body
-  })
+  return platform
+}
+
+const scrapeMudletSnapshotsForLinks = async prNumber => {
```

Comment:
Perhaps not scrape anymore?

Reply:
Argh, right. I knew I forgot something... The version here is already running, but I'll fix it ASAP, but I don't know if I get to it before the vacation.